scipy/spatial/ckdtree.pyx

```diff
@@ -613,6 +608,26 @@ cdef public class cKDTree [object ckdtree, type ckdtree_type]:
             Number of jobs to schedule for parallel processing. If -1 is given
             all processors are used. Default: 1.
                         
+        Attributes
+        ----------
+        data : array_like, shape (n,m)
```

Comment:
Should be ``ndarray`` instead of ``array_like``. ``array_like`` includes things like lists; if you pass a list in then an array is created from it by ``ascontiguousarray`` before storing ``self.data``